packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -21,7 +23,7 @@ const double _kMenuMaxWidth = 5.0 * _kMenuWidthStep;
 const double _kMenuHorizontalPadding = 16.0;
 const double _kMenuVerticalPadding = 8.0;
 
-class _PopupMenuPainter extends CustomPainter {
+class _PopupMenuPainter extends CustomMaterialPainter {
```

Comment:
This is a real shame.  You have to spill a lot of your guts to support this use case.  Maybe we should figure out if we can get menus working with normal Material containers using overflow box or some other of our fancy new technology.